layout/header/header-about/index.js

```diff
@@ -1,11 +1,11 @@
 import { connect } from 'react-redux';
 import * as actions from '../header-actions';
 
-import HeaderAboutComponent from './component';
+import HeaderGetInvolvedComponent from './component';
```

Comment:
You changed the about component, didn't you? `layout/header/header-about/index.js`

Reply:
Good spot! Thanks!